Userland/Applications/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ add_subdirectory(NetworkSettings)
 add_subdirectory(PartitionEditor)
 add_subdirectory(PDFViewer)
 add_subdirectory(Piano)
+add_subdirectory(PixelFlutClient)
```

Comment:
This is the only application living in `Userland/Applications` without a GUI, maybe this is better suited as a small(ish) utility?

Reply:
I think it's cleaner in more than one file, but I can squash them together since you're correct about it; there's also no intent of the client having a GUI at all.

Reply:
I know that our utilities are all single cpp files, but I don't think we have a rule that they have to be? I wouldn't mind having them in a subdirectory.